### PR TITLE
Foreman

### PR DESCRIPTION
**Pull request by dandds**
This adds foreman for managing all the services. It's a necessary step along the way to end-to-end testing. It's contingent on [this PR](https://github.com/dod-ccpo/authnid/pull/12) over in authnid. 

If you want to try it before the authnid PR is merged:

- run `script/setup`
- remove the `-d` flag from line 8 of `services/authnid/script/up`

Note that, for now, `script/test` runs all the individual service test suites. On a subsequent PR it will be responsible for running real integration tests.

### CODE REVIEW

File: script/server [hunk 1]
```diff
@@ -1,7 +1,12 @@
...
+eval "$(cat services.ini  | ./ini_parser.py)"
 
-source services.ini
+rm -f Procfile || true
+touch Procfile
 
-echo "not yet implemented"
+for service in ${SERVICE_NAMES[@]}; do
+    echo "$service: ./services/$service/script/server" >> Procfile
+done
```
Comment (briandds) [inline script/server]: It seems like maybe this should happen in `bootstrap` or `setup`

**Review by briandds — CHANGES_REQUESTED** [review]

**Comment by dandds** [comment]
New commits address Brian's suggestion and add a quick script for reaping rogue services. (Sometimes if something goes wrong with foreman, it doesn't seem to kill all the child processes.)

**Comment by dandds** [comment]
Reverting the reap script until we figure out something better.

**Review by briandds — APPROVED** [review]
Looks good!

**Pull request merged**